api/v1/client.go

```diff
@@ -1437,19 +1438,21 @@ func (c *storageClient) Validators(ctx context.Context, r *http.Request) (*Valid
 		%s.accounts.escrow_balance_active AS escrow,
 		%s.commissions.schedule AS commissions_schedule,
 		case when exists(select null from %s.nodes where %s.entities.id = %s.nodes.entity_id AND voting_power > 0) then true else false end as active,
-		case when exists(select null from %s.nodes where %s.entities.id = %s.nodes.entity_id AND %s.nodes.roles like 'validator') then true else false end as status,
+		case when exists(select null from %s.nodes where %s.entities.id = %s.nodes.entity_id AND %s.nodes.roles like '%%validator%%') then true else false end as status,
 		%s.entities.meta AS meta
 	FROM %s.entities
 	JOIN %s.accounts ON %s.entities.address = %s.accounts.address
-	JOIN %s.commissions ON %s.entities.address = %s.commissions.address
+	LEFT JOIN %s.commissions ON %s.entities.address = %s.commissions.address
 	JOIN %s.nodes ON %s.entities.id = %s.nodes.entity_id
+		AND %s.nodes.roles like '%%validator%%'
 		AND %s.nodes.voting_power = (
 			SELECT max(voting_power)
 			FROM %s.nodes
 			WHERE %s.entities.id = %s.nodes.entity_id
-				AND %s.nodes.roles like 'validator'
+				AND %s.nodes.roles like '%%validator%%'
 		)
-	`, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID, chainID), c.db)
+	ORDER BY voting_power DESC
```

Comment:
This is different from `escrow`.

Reply:
I guess either ordering should work because this is really only problematic for the ones at the bottom where `voting_power == 0`.